JavaScriptSDK/AppInsightsCore.ts

```diff
@@ -260,21 +263,34 @@ class ChannelController implements ITelemetryPlugin {
 
     identifier: string = "ChannelControllerPlugin";

```

Comment:
How the initialization code will look when using this?, do you have a test page with this?

Reply:
no change, behavior remains same as before. pass in channel as you would do so previously in array to core. You can also give an input through config.